extensions/shared/external-media/media-browser/index.js

```diff
@@ -37,8 +58,11 @@ function MediaBrowser( props ) {
 		onCopy,
 	} = props;
 	const [ selected, setSelected ] = useState( [] );
+	const [ focused, setFocused ] = useState( -1 );
```

Comment:
I'm not sure what it should be named, but I'm worried calling this `focused` and `setFocused` indicates we're trying to keep track of what media item has focus. We don't actually care to keep track of the currently focused item. We only care about _sending focus to an item in some contexts_:

- Arrow key press
- Load More button click

The rest of the contexts (tab, click, etc) are already handled by the browser and we don't need to try to track or update those. I think we should rename this to something that is more in-line with what the state is doing: Sending Focus to a new item. Ideas:

- `sendFocusTo`: Accurate, but sounds like a method
- `nextFocus`: Accurate, but could have the same issue as above
- `navigateTo`: Similar to `sendFocusTo` but maybe sounds less like a method?

None of these feel great though. Any suggestions?